File: University-Courseworks/cs1ip/coursework1/volume.py
import math

def Volume(diameter):
    radius = (1 / 2.0) * float(diameter) # Diameter = 2 x Radius
    volume = (4.0 / 3.0) * math.pi * math.pow(float(radius), 3) # Sphere's Volume
    return float(volume)

if __name__ == "__main__":
    print(Volume(20.24))

File: CS1IP/cs1ip/coursework1/volume.java
public class Volume {
    public static double sphere(double diameter) {
        double radius = (1 / 2.0) * diameter;
        double volume = (4.0 / 3.0) * Math.PI * Math.pow(radius, 3);
        return volume;
    }

    public static void main(String[] args) {
        System.out.println(sphere(20.24));
    }
}
